docs/setup/README.md

```diff
@@ -0,0 +1,334 @@
+# Setup
+
+This document goes through the steps you need to take to set up a token gateway.
+
+## Prerequisites
+
+You need:
+
+* a connection to nodes that are connected to origin and auxiliary (for help see [how to connect]('../test_net/README.md))
+* OST on auxiliary to deploy contracts
+* an EIP20 token deployed on origin
+* a utility token deployed on auxiliary
+* optional: you already have `Organization`s deployed on origin and auxiliary
+* optional: you already have state root providers (e.g. `Anchor`s) deployed on origin and auxiliary
+
+The following is required to use mosaic.js setup:
+
+```js
+const { Setup } = require('@openstfoundation/mosaic.js');
+const Web3 = require('web3');
+
+const origin = {
+  web3: new Web3('http://localhost:8545'),
+  chainId: '3',
+  // The address that you will use to deploy contracts:
+  deployer: '0x6d4e5f5ca54b88fdb68a5d0e6ea5aa3869f88116',
+  txOptions: {
+    gasPrice: '13000000000', // 13 GWei
+  },
+  // The address of your EIP20 token:
+  token: '0x2902a9E437c07168fc73B96821ed82b8eB69B20F',
+  // The address of OST (you can use this one):
+  baseToken: '0xca954C91BE676cBC4D5Ab5F624b37402E5f0d957',
+  // For now, burnt tokens are sent to the zero address
+  burner: '0x0000000000000000000000000000000000000000',
+  // The address of your organization (owner):
+  organization: '0x6d4e5f5ca54b88fdb68a5d0e6ea5aa3869f88116',
+};
+const auxiliary = {
+  web3: new Web3('http://localhost:8546'),
+  chainId: '1409',
+  // The address that you will use to deploy contracts:
+  deployer: '0x48dbe0b823ba5e4a3114925242b75c5ae2041f62',
+  txOptions: {
+    gasPrice: '1000000000', // 1 GWei
+  },
+  // The address of the utility token to your EIP20 token:
+  utilityToken: '0x10f7af624395843eF6d5c1ca2272B648C35b5B94',
+  // For now, burnt tokens are sent to the zero address
+  burner: '0x0000000000000000000000000000000000000000',
+  // The address of your organization (owner):
+  organization: '0x48dbe0b823ba5e4a3114925242b75c5ae2041f62',
+};
+```
+
+## Organizations
+
+Organizations are used for access control to specific methods of contracts.
+
+If you don't have any organizations deployed, yet, mosaic.js can help you to deploy organizations
+on origin and auxiliary:
+
+```js
+const [originOrganization, auxiliaryOrganization] = await Setup.organizations(
+  origin.web3,
+  auxiliary.web3,
+  {
+    deployer: origin.deployer,
+    // The owner of an organization contract has all privileges:
+    owner: origin.organization,
```

Comment:
`origin.organization` is confusing in setup of `Organization`; perhaps rename the member `origin.masterKey`